src/Files.App/BaseLayout.cs

```diff
@@ -453,9 +487,11 @@ public void SetSelectedItemsOnNavigation()
 		{
 			try
 			{
-				if (navigationArguments is not null && navigationArguments.SelectItems is not null && navigationArguments.SelectItems.Any())
+				if (navigationArguments is not null &&
+					navigationArguments.SelectItems is not null &&
+					navigationArguments.SelectItems.Any())
 				{
-					List<ListedItem> liItemsToSelect = new List<ListedItem>();
+					List<ListedItem> liItemsToSelect = new();
```

Comment:
Is there any practical difference?

Reply:
- Remove duplication and redundancy (if that were `ObservableCollection<FolderCardItemViewModel> folderCardItems = newObservableCollection<FolderCardItemViewModel>();`, it might be redundant)
- IntelliSense suggested

Reply:
Got it, thanks